cmd/dartboard/subcommands/deploy.go

```diff
@@ -188,31 +199,32 @@ func chartInstallRancher(r *dart.Dart, rancherImageTag string, cluster *tofu.Clu
 	rancherClusterName := clusterAdd.Public.Name
 	rancherClusterURL := clusterAdd.Public.HTTPSURL
 
-	chartVals := getRancherValsJSON(r.ChartVariables.RancherImageOverride, rancherImageTag, r.ChartVariables.AdminPassword, rancherClusterName, rancherClusterURL, r.ChartVariables.RancherReplicas)
-
-	var extraArgs []string
-	if r.ChartVariables.RancherValues != "" {
-		p, err := writeValuesFile(r.ChartVariables.RancherValues)
-		if err != nil {
-			return fmt.Errorf("writing extra values file: %w", err)
-		}
-		defer os.Remove(p)
-
-		extraArgs = append(extraArgs, "-f", p)
+	var extraEnv []map[string]any
+	extraEnv = []map[string]any{
+		{
+			"name":  "CATTLE_SERVER_URL",
+			"value": rancherClusterURL,
+		},
+		{
+			"name":  "CATTLE_PROMETHEUS_METRICS",
+			"value": "true",
+		},
+		{
+			"name":  "CATTLE_DEV_MODE",
+			"value": "true",
+		},
```

Comment:
Do we want a `CATTLE_AGENT_IMAGE` extra env var too? For https://github.com/rancher/dartboard/issues/34

Reply:
So #34 should be handled by PR/#54

Reply:
I see, just by what's passed into the `extra_environment_variables` directly. Sounds good, we don't need any default!